packages/editor-ui/src/api/helpers.ts

```diff
@@ -91,6 +91,7 @@ export async function makeRestApiRequest(context: IRestApiContext, method: Metho
 	return response.data;
 }
 
-export async function get(baseURL: string, endpoint: string, params?: IDataObject) {
-	return await request({method: 'GET', baseURL, endpoint, data: params});
+export async function get(baseURL: string, endpoint: string, params?: IDataObject, instanceId?: string) {
+	const headers = {[INSTANCE_ID_HEADER as string] : instanceId};
+	return await request({method: 'GET', baseURL, endpoint, headers, data: params});
```

Comment:
since this is a generic function (that's used in other places), this will break existing use in other places
```suggestion
export async function get(baseURL: string, endpoint: string, params?: IDataObject, headers?: IDataObject) {
	return await request({method: 'GET', baseURL, endpoint, headers, data: params});
```